setup.py

```diff
@@ -1,27 +1,45 @@
-#!/usr/bin/env python
 from setuptools import setup, find_packages
 
-setup(name='django-oscar-api',
-      version='0.1',
-      url='https://github.com/tangentlabs/django-oscar-api',
-      author="David Winterbottom",
-      author_email="david.winterbottom@tangentlabs.co.uk",
-      description="REST API module for django-oscar",
-      long_description=open('README.rst').read(),
-      keywords="Oscar, REST, API",
-      license='BSD',
-      packages=find_packages(exclude=['sandbox*', 'tests*']),
-      include_package_data=True,
-      install_requires=[
-          'django-oscar>=0.4',
-      ],
-      # See http://pypi.python.org/pypi?%3Aaction=list_classifiers
-      classifiers=[
-          'Development Status :: 4 - Beta',
-          'Environment :: Web Environment',
-          'Framework :: Django',
-          'Intended Audience :: Developers',
-          'License :: OSI Approved :: BSD License',
-          'Operating System :: Unix',
-          'Programming Language :: Python']
-      )
+
+__version__ = "0.0.13"
+
+
+setup(
+    # package name in pypi
+    name='django-oscar-api',
+    # extract version from module.
+    version=__version__,
+    description="REST API module for django-oscar",
+    long_description=open('README.rst').read(),
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Environment :: Web Environment',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Operating System :: Unix',
+        'Programming Language :: Python'
+    ],
+    keywords='',
+    author='Lars van de Kerkhof, Martijn Jacobs',
+    author_email='lars@permanentmarkers.nl, martijn@devopsconsulting.nl',
+    url='https://github.com/tangentlabs/django-oscar-api',
+    license='BSD',
+    # include all packages in the egg, except the test package.
+    packages=find_packages(exclude=['ez_setup', 'examples', 'tests', 'sandbox']),
+    # for avoiding conflict have one namespace for all apc related eggs.
+    namespace_packages=[],
+    # include non python files
+    include_package_data=True,
+    zip_safe=False,
+    # specify dependencies
+    install_requires=[
+        'setuptools',
+        'django-oscar',
```

Comment:
@codeinthehole lower the version? i see 0.0.13 on top is that Oscar's version or i'm too sleepy? :)